NumericOutcome.md

```diff
@@ -66,8 +66,7 @@ up in the case of a tie.
 
 #### Rounding Interval Serialization
 
-1. type: 42788 (`rounding_intervals_v0`)
-2. data:
+1. data:
    * [`u16`:`num_rounding_intervals`]
```

Comment:
@Tibo-lg  Is this consistent with your implementation?

Reply:
Addressed in 70032e32d69689547ed9a96faa8e366bbf039cd5

Reply:
I think there is another bug here, I believe this is prefixed by a `bigsize` in your implementation @Tibo-lg rather than a `u16`.

Reply:
Yes in that case I should have updated the spec to be consistent with other collection prefixes. Updated now.